src/content/docs/zh-cn/features/dialog.mdx

```diff
@@ -0,0 +1,292 @@
+    />
+
+</TabItem>
+</Tabs>
+
+## 用法
+
+对话框插件可以在 JavaScript 和 Rust 中使用。你可以这样使用它：
+
+在 JavaScript:
+
+- [创建 Yes/No 对话框](#创建-yesno-对话框)
+- [创建 Ok/Cancel 对话框](#创建-okcancel-对话框)
+- [创建 Message 对话框](#创建-message-对话框)
+- [打开一个文件选择对话框](#打开一个文件选择对话框)
+- [保存到文件对话框](#保存到文件对话框)
+
+在 Rust:
+
+- [建立一个询问对话框](#建立一个询问对话框)
+- [建立消息对话框](#建立消息对话框)
+- [建立一个文件选择器对话框](#建立一个文件选择器对话框)
+
+---
+
+### JavaScript
+
+可以在 JavaScript API 参考中查看所有 [Dialog 选项](/references/v2/js/dialog/)。
+
+{/* ASK */}
+
+#### 创建 Yes/No 对话框
+
+显示一个带有 “Yes” 和 "No" 按钮的提问对话框。
+
+```js
+import { ask } from '@tauri-apps/plugin-dialog';
+
+// 创建 Yes/No 对话框
+const answer = await ask('This action cannot be reverted. Are you sure?', {
+	title: 'Tauri',
+	type: 'warning',
+});
+
+console.log(answer);
+// Prints boolean to the console
+```
+
+{/* CONFIRM */}
+
+#### 创建 Ok/Cancel 对话框
+
+显示一个带有 “Ok” 和 "Cancel" 按钮的提问对话框。
+
+```js
+import { confirm } from '@tauri-apps/plugin-dialog';
+
+// Creates a confirmation Ok/Cancel dialog
+const confirmation = await confirm(
+	'This action cannot be reverted. Are you sure?',
+	{ title: 'Tauri', type: 'warning' }
+);
+
+console.log(confirmation);
+// Prints boolean to the console
+```
+
+{/* MESSAGE */}
+
+#### 创建 Message 对话框
+
+一个带有 “Ok” 按钮的消息对话框。请注意，如果用户关闭对话框，它将返回 `false`。
+
+```js
+import { message } from '@tauri-apps/plugin-dialog';
+
+// Shows message
+await message('File not found', { title: 'Tauri', type: 'error' });
+```
+
+{/* OPEN */}
+
+#### 打开一个文件选择对话框
+
+Open a file/directory selection dialog.
+
+打开一个文件/目录选择对话框。
+
+`multiple` 选项控制对话框是否允许多重选择，而 `directory` 则控制对话框是否允许目录选择。
+
+```js
+import { open } from '@tauri-apps/plugin-dialog';
+
+// Open a dialog
+const file = await open({
+	multiple: false,
+	directory: false,
+});
+console.log(file);
+// Prints file path and name to the console
+```
+
+{/* SAVE */}
+
+#### 保存到文件对话框
+
+打开一个文件/目录保存对话框。
+
+```js
+import { save } from '@tauri-apps/plugin-dialog';
+// Prompt to save a 'My Filter' with extension .png or .jpeg
+const path = await save({
+	filters: [
+		{
+			name: 'My Filter',
+			extensions: ['png', 'jpeg'],
+		},
+	],
+});
+console.log(path);
+// Prints the chosen path
+```
+
+---
+
+### Rust
+
+请参阅 [Rust API 参考](https://docs.rs/tauri-plugin-dialog/)以查看所有可用选项。
+
+#### 建立一个询问对话框
+
+显示一个带有 “Absolutely” 和 “Totally” 按钮的问题对话框。
+
+```rust
+use tauri_plugin_dialog::DialogExt;
+
+let answer = app.dialog()
+        .message("Tauri is Awesome")
+        .title("Tauri is Awesome")
+        .ok_button_label("Absolutely")
+        .cancel_button_label("Totally")
+        .blocking_show();
+```
+
+如果你需要一个非阻塞操作，你可以使用 `show()`：
+
+```rust
+use tauri_plugin_dialog::DialogExt;
+
+app.dialog()
+    .message("Tauri is Awesome")
+    .title("Tauri is Awesome")
+    .ok_button_label("Absolutely")
+    .cancel_button_label("Totally")
+    .show(|result| match result {
+        true => // do something,
+        false =>// do something,
+    });
+```
+
+#### 建立消息对话框
+
+一个带有 “Ok” 按钮的消息对话框。请注意，如果用户关闭对话框，它将返回 `false`。
```

Comment:
```suggestion
一个带有 "Ok" 按钮的消息对话框。请注意，如果用户关闭对话框，它将返回 `false`。
```
Formatting nitpick. Let's stick to plain double quotation mark for now for easier consistency.